### PR TITLE
feat: Setup Navbar

### PR DESCRIPTION
**Pull request by ronald-luo**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,9 +1,14 @@
 import { useState } from "react";
```
Comment (Krish120003) [inline src/App.tsx:1]: Why do we have this import?

File: src/App.tsx [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+import Navigation from "./components/navigation/navigation";
 import LandingPage from "./views/Landing";
+import clouds from "../public/Clouds.svg";
 
 function App() {
+  const [count, setCount] = useState(0);
```
Comment (Krish120003) [inline src/App.tsx:7]: We don't need any state here.

File: src/index.css [hunk 3]
```diff
@@ -1,36 +1,38 @@
...
+        #47f4ff 46.35%,
+        rgba(0, 123, 186, 0.66) 58.85%,
+        #03476d 71.87%,
+        #000000 90.1%);
 
     background-size: 400% 400%;
     animation: gradient 15s ease infinite;
   }
+
   .mBG {
```
Comment (Krish120003) [inline src/index.css:25]: Why is this here? Why are we not using Tailwind for this?

File: src/index.css [hunk 4]
```diff
@@ -1,36 +1,38 @@
...
     background: #606060;
     box-shadow: inset -5px -5px 0px rgba(49, 49, 49, 0.5),
       inset 5px 5px 0px #a9a9a9;
     width: auto;
     height: 100%;
   }
+
   .mBG:hover {
```
Comment (Krish120003) [inline src/index.css:33]: Same here - why not tailwind?

**Review by Krish120003 — CHANGES_REQUESTED** [review]

**Review by KaiKitJeffreyChan — APPROVED** [review]

**Review by Krish120003 — APPROVED** [review]
